release_tester/license_manager_tests/base/single_server_base.py

```diff
@@ -0,0 +1,62 @@
+"""License manager tests: single server"""
+import json
+
+# pylint: disable=import-error
+from arangodb.installers import RunProperties
+from arangodb.instance import InstanceType
+from arangodb.starter.deployments import make_runner, RunnerType
+from license_manager_tests.base.license_manager_base_test_suite import LicenseManagerBaseTestSuite
+from reporting.reporting_utils import step
+from test_suites_core.base_test_suite import run_before_suite
+
+
+class LicenseManagerSingleServerBaseTestSuite(LicenseManagerBaseTestSuite):
+    """License manager tests: single server"""
+
+    def get_default_instance_type(self):
+        """get the instance type we should communicate with"""
+        return InstanceType.SINGLE
+
+    @run_before_suite
+    def start(self):
+        """start a single server setup before running tests"""
+        self.start_single_server()
+
+    @step
+    def start_single_server(self):
+        """start a single server setup"""
+        self.runner = make_runner(
+            runner_type=RunnerType.SINGLE,
+            abort_on_error=False,
```

Comment:
BTW, why `abort_on_error` is always `False` for all inctances?

Reply:
We do not want the runner to decide if we want to abort and exit. This must be decided in test case code. Even if an assertion failed in one test case, we might want to continue with other test cases.